cpg-concepts/src/main/kotlin/de/fraunhofer/aisec/cpg/graph/concepts/auth/Authorization.kt

```diff
@@ -25,12 +25,5 @@
  */
 package de.fraunhofer.aisec.cpg.graph.concepts.auth
 
-import de.fraunhofer.aisec.cpg.graph.Node
-import de.fraunhofer.aisec.cpg.graph.concepts.Concept
-
-/**
- * Represents an abstract concept of `authorization`.
- *
- * @param underlyingNode The underlying CPG node.
- */
-open class Authorization(underlyingNode: Node? = null) : Concept(underlyingNode = underlyingNode)
+/** Represents an abstract concept of `authorization`. */
+data class Authorization(val authorizationId: String, val permissions: Set<String> = emptySet())
```

Comment:
I still want to keep the inheritance to the `Concept` class. You need to change the underlying `Concept` class so that it does not need a mandatory `underlyingNode` parameter anymore